src/ray/raylet/design_docs/task_states.rst

```diff
@@ -0,0 +1,36 @@
+A task can be in one of the following states:
```

Comment:
Can you add something like this to the top of the file? (this is normal for rst files)

```
Task States
===========
```